code/obj/item/kitchen.dm

```diff
@@ -681,7 +692,7 @@ TRAYS
 			src.remove_contents(pick(src.contents))
 
 	attack(mob/M, mob/user)
-		if(user.a_intent == INTENT_HARM)
+		if(user.a_intent == INTENT_HARM && src.is_plate)
```

Comment:
why?

Reply:
because if i don't specify that it's a plate, boxes will have the same behavior. but i'll move this condition check to the pizza box code.

Reply:
yes, why not?

Reply:
i don't really get what you mean.

Reply:
oh, you might be trying to say "why not have the same behavior?", and the answer is simple: plates smash into plate bits. boxes don't.